src/contracts/common/ExecutionBase.sol

```diff
@@ -97,16 +97,11 @@ contract Base {
     }
 
     modifier onlyIfFirstDonationInCurrentPhase() {
-
-        // Get current phase from lockState
-
-        // if(phasesWithDonations & _lockState() == 0)
-
-        // phasesWithDonations |= uint16(1<<(EXECUTION_PHASE_OFFSET + uint16(phase)));
-
-        // if (_makingPayments()) {
-        //     revert("ERR-EV014 NotFirstDonation");
-        // }
+        uint16 currentPhase = SafetyBits.getCurrentExecutionPhase(_lockState());
```

Comment:
Not sure if this storage is trustable - there's a delegatecall in between the storage write and the storage read. the dapp might have defenses against being exploited, but we can't assume that we'll inherit those defenses.